docs/zh_cn/05-supported-backends/onnxruntime.md

```diff
@@ -1,42 +1,42 @@
 # onnxruntime 支持情况
 
-## Introduction of ONNX Runtime
+## ONNX Runtime 介绍
 
-**ONNX Runtime** is a cross-platform inference and training accelerator compatible with many popular ML/DNN frameworks. Check its [github](https://github.com/microsoft/onnxruntime) for more information.
+**ONNX Runtime** 是一个跨平台的推理和训练加速器，与许多流行的ML/DNN框架兼容。查看其[github](https://github.com/microsoft/onnxruntime)以获取更多信息。
 
-## Installation
+## 安装
 
-*Please note that only **onnxruntime>=1.8.1** of on Linux platform is supported by now.*
+*请注意，目前Linux平台只支持 **onnxruntime>=1.8.1** 。*
```

Comment:
We can build mmdeploy with onnxruntime on Linux/Windows/Mac. So we could remove the platform restriction notes.